_field-types/mapping-parameters/ignore-malformed.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Ignore_malformed
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 45
+has_children: false
+has_toc: false
+---
+
+# ignore_malformed
+
+The `ignore_malformed` mapping parameter instructs the indexing engine to ignore values that do not match the field’s expected format. When enabled, malformed values are not indexed, preventing entire document rejection due to data format issues. This ensures that documents are still stored even if one or more fields contain data that cannot be parsed.
```

Comment:
```suggestion
The `ignore_malformed` mapping parameter instructs the indexing engine to ignore values that do not match the field’s expected format. When enabled, malformed values are not indexed, preventing entire-document rejection because of data format issues. This ensures that documents are still stored even if one or more fields contain data that cannot be parsed.
```